### ISSUE TITLE
Make placeholder for select input disabled

### ISSUE THREAD

**Issue by willishq** [spatie/laravel-html#95]
When I am creating a form with a placeholder for the select input, it currently makes the placeholder selectable. What I am suggesting is that we make the placeholder option disabled so that once it has been changed, the placeholder can not be selected again. This is especially useful for required form elements as it stops the user from being able to deselect an option and invalidating the form.

My suggestion would be to make the placeholder default to being selectable and adding a new flag to the placeholder method, `disabled` which defaults to false:

```php
html()->select('option', $options)->placeholder('Select One', false); // <option>Select One</option>
html()->select('option', $options)->placeholder('Select One', true); // <option disabled>Select One</option>
```

I do, however, have reservations about having a `true` value representing disabling an element and think the feature could benefit from discussion.

If everybody is in agreement I can create a pull request for these changes in the next few days.

**Comment by sebastiandedeyne** [comment]
I agree with this behaviour, and would rather do it by default in the next major version, but I don't think we should push this as a breaking change now.

How about we add a `disabledPlaceholder` method for now? That way it's more explicit than a boolean value. Feel free to send a PR if you agree 👍
